android/assets/jsons/translations/French.properties

```diff
@@ -739,13 +739,13 @@ Closest city = Ville la plus proche
 Action = Action
 Defeated = Vaincu
  # Requires translation!
-[numberOfCivs] Civilizations in the game = 
+[numberOfCivs] Civilizations in the game = [numberofCivs] Civilisation dans la partie 
  # Requires translation!
-Our Civilization: = 
+Our Civilization: = Notre civilisation 
  # Requires translation!
-Known and alive ([numberOfCivs]) = 
+Known and alive ([numberOfCivs]) = Connues et vivantes ([numberofCivs]) 
```

Comment:
`Known and alive ([numberOfCivs]) = Connues et vivantes ([numberOfCivs]) `

case-sensitive